app/presenters/publishing_api_presenters/case_study.rb

```diff
@@ -27,6 +27,7 @@ def details
       format_display_type: item.display_type_key,
       first_public_at: first_public_at,
       change_history: item.change_history.as_json,
+      change_note: item.most_recent_change_note,
```

Comment:
I assume the most recent item in the change_history would be the same as the change_note?